<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmLogin</class>
 <widget class="QStackedWidget" name="FrmLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>750</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);

</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Panel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>5</number>
  </property>
  <property name="currentIndex">
   <number>1</number>
  </property>
  <widget class="QWidget" name="page1">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 85, 255);</string>
   </property>
   <widget class="QLabel" name="lblLoginUser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Iniciar sesión</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginUserIcon">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneLoginUser">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneLoginPassword">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblLoginPassword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>439</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>580</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Acceder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGoRegister">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>710</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ir a registrarse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoginTogglePass">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="page2">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="btnGoBack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblRegisterTitle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>15</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Registrar usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRegisterUser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>109</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Usuario:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneRegisterUser">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRegisterPassword">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>199</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneRegisterPassword">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>299</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSurname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>399</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Apellido: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lneSurname">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblPhone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>499</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Teléfono:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnePhone">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>530</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrarse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegisterTogglePass">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
